packages/drivers/redis-driver/src/index.ts

```diff
@@ -12,12 +12,10 @@ import { Query } from './Query';
 import { RoomData } from './RoomData';
 
 export class RedisDriver implements MatchMakerDriver {
-  private readonly _client: RedisClient;
-  private readonly hgetall: (key: string) => Promise<{ [key: string]: string }>;
+  private readonly _client: Redis.Redis;
 
-  constructor(options?: ClientOpts, key: string = 'roomcaches') {
-    this._client = redis.createClient(options);
-    this.hgetall = promisify(this._client.hgetall).bind(this._client);
+  constructor(options?: Redis.RedisOptions, key: string = 'roomcaches') {
+    this._client = new Redis(options);
```

Comment:
Don't we need to use the [`Cluster`](https://github.com/luin/ioredis#cluster) interface instead to handle clusters? Connecting to a Redis cluster configuration endpoint with `ioredis.Redis` seems to throw `ReplyError: MOVED`.

Reply:
Here's a PR for cluster support in the proxy: https://github.com/colyseus/proxy/pull/25